guides/docs/phoenix_mix_tasks.md

```diff
@@ -30,9 +30,9 @@ Note that there are still other tasks under the full phoenix name:
 ```console
 ➜  mix help | grep -i phoenix
 mix local.phoenix      # Updates Phoenix locally
-mix phoenix.gen.html   # Generates controller, model and views for an HTML based resource
-mix phoenix.new        # Creates a new Phoenix v1.2.1 application
-mix phoenix.server     # Starts applications and their servers
+mix phx.gen.html   # Generates controller, model and views for an HTML based resource
+mix phx.new        # Creates a new Phoenix v1.2.1 application
```

Comment:
We should probably remove the version here so we don't need to keep updating it.

Reply:
Actually, we need to keep these as is, since the section states:

 >  Note that there are still other tasks under the full phoenix name:

Reply:
Maybe we just kill the entire section because the only one that remains on my machine is `phoeinx.new` and `local.phoenix` thats because I have it laying around from previous versions.

Reply:
@jeregrine The others exist when you are in an application using Phoenix. Phoenix 1.3 supports both sets of generators, however the phoenix.* functions are gone in 1.4